packages/cursorless-engine/src/actions/actions.types.ts

```diff
@@ -124,7 +124,11 @@ export interface ActionRecord extends Record<SimpleActionName, SimpleAction> {
   };
 
   generateSnippet: {
-    run(targets: Target[], snippetName?: string): Promise<ActionReturnValue>;
+    run(
+      targets: Target[],
+      dirPath?: string,
```

Comment:
Not thrilled about `dirPath` but it makes sense after discussion -- we sometimes have the directory name `dirName` so it's needed to disambiguate.

If we had a `Path` type of class it would be nicer so we didn't have to put the type information in the name itself, but we don't. We could make a wrapper ourselves or just do two type aliases. We might make a followup for this.

`storageDirectory` would be my choice for the name but not if it has to be `storageDirectoryPath`, I think that's worse than the status quo

Reply:
Settled on `directory` for now